src/components/graph-crumb/GraphCrumb.jsx

```diff
@@ -76,7 +76,8 @@ const NodeUnderline = ({ cid, children }) => {
 }
 
 const Path = ({ path, hrefBase, sourceCid }) => {
-  const parts = path.split('/').filter(p => !!p)
+  const parts = [path]
+
```

Comment:
@lidel @hacdias this could use your view. If you dive into a deeply nested component where some have child node paths like "Links/0" or in the git block, like "parent/0" etc.. then the individual sections of that child-node path name are clickable, which ends up not being useful in the breadcrumbs. I updated this so that only renderable paths are clickable in the breadcrumbs. lmk if i'm missing some usecase.